acceptance_tests/utilities/pubsub_helper.py

```diff
@@ -70,7 +63,6 @@ def _pull_exact_number_of_messages(subscriber, subscription_path, expected_msg_c
     # Synchronously pull messages one at at time until we either hit the expected number or the timeout passes.
     received_messages = []
     deadline = time.time() + timeout
-
```

Comment:
Space was here for readability of code and comments, don't think we want to stop doing that?